picard/ui/mainwindow.py

```diff
@@ -1497,8 +1497,8 @@ def make_script_selector_menu(self):
         def _add_menu_item(title, id):
             script_action = QtWidgets.QAction(title, self.script_quick_selector_menu)
             script_action.triggered.connect(partial(self.select_new_naming_script, id))
-            if id == selected_script_id:
-                script_action.setIcon(self.style().standardIcon(QtWidgets.QStyle.SP_DialogApplyButton))
+            script_action.setCheckable(True)
+            script_action.setChecked(id == selected_script_id)
```

Comment:
Actually since those are exclusive, it should be radio buttons

See https://github.com/rdswift/picard/pull/8

Reply:
Much better looking as well.  Thanks.